pip_outdated/__init__.py

```diff
@@ -29,6 +34,13 @@ def main():
     from .check_outdated import check_outdated
     from .print_outdated import print_outdated
     requires = find_require(args.file)
-    outdated_results = check_outdated(requires)
-    print_outdated(outdated_results, args.quiet)
+    
+    import aiohttp
+    headers = {"User-Agent": "pip-outdated"}
+    async with aiohttp.ClientSession(headers=headers) as session:
```

Comment:
I'd suggest to add a rate limit here, to avoid bombarding pypi too much with simultaneous requests. This can be easily added by specifying a connector:

```python
async with aiohttp.ClientSession(headers=headers, connector=aiohttp.TCPConnector(limit=10)) as session:
```